### DISCUSSION TITLE
Where some fixes in `main` are going be included?

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [python-poetry/poetry#9793] [Q&A] [answered]
I'm looking for the release including the fix: https://github.com/python-poetry/poetry-core/pull/722
It was merged into `main`, but didn't landed in 1.9.1. Is it possible to cherry-pick it into the next poetry-core release please?

**Comment by radoering** [comment, marked as answer]
It will be included in poetry-core 2.0, which will be released shortly before poetry 2.0. The progress is tracked in https://github.com/python-poetry/poetry/issues/9448.

It is too risky to include the fix in a patch release. In other words: If we do a 1.9.2 release for some reason (which is not planned by the way), it will not be included.

**Reply by FeodorFitsner** [reply]
Thank you! Subscribed to that issue.

Is there a rough estimate when 2.0 might be released?

**Reply by radoering** [reply]
Hopefully this year. However, it is difficult to estimate because it not only depends on my own availability but also on the availability of other maintainers.
